tests/integrations/server_encryption.rs

```diff
@@ -13,6 +13,7 @@ fn test_snapshot_encryption<T: Simulator>(cluster: &mut Cluster<T>) {
     }
 
     cluster.pd_client.must_add_peer(r1, new_learner_peer(2, 2));
+    cluster.pd_client.must_add_peer(r1, new_peer(2, 2));
```

Comment:
If there is only one voter, add an another voter or learner don't need the target peer is up to date.
So please write a `for loop` to stabilize it.

Reply:
I think we can add an extra `cluster.must_put` to ensure peer 2 have all value, because after promoting peer 2 to a voter, peer 2 must in the majority of the raft group, so the leader must replicate the logs (and including all previous logs) to peer 2 before committing it